CHANGELOG.md

```diff
@@ -34,6 +34,7 @@ Please add to the relevant subsections under Unreleased below on every PR where
 
 ### Fixed
 
+- Fix process stalling when trying to call `artifact.download()` when `WANDB_MODE = offline` in https://github.com/wandb/wandb/pull/8009
```

Comment:
you are adding it a released change log, please move it to the unreleased section.
Also i think it falls under the Changed see [here](https://keepachangelog.com/en/1.1.0/#how)

Reply:
Done